tracer/src/Datadog.Trace/ContinuousProfiler/EnvironmentVariables.cs

```diff
@@ -0,0 +1,13 @@
+// <copyright file="EnvironmentVariables.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ContinuousProfiler
+{
+    internal static class EnvironmentVariables
+    {
+        public const string ProfilingEnabled = "DD_PROFILING_ENABLED";
+        public const string CodeHotspotEnabled = "DD_PROFILING_CODEHOTSPOT_ENABLED";
```

Comment:
[branding] I expect the feature is called "Code Hotspots" in plural.

```suggestion
        public const string CodeHotspotsEnabled = "DD_PROFILING_CODEHOTSPOTS_ENABLED";
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2587/commits/88be457a341944a87f8b4129fb0e320cba63e36a